rclpy/src/rclpy/_rclpy.c

```diff
@@ -3423,17 +3423,8 @@ rclpy_shutdown(PyObject * Py_UNUSED(self), PyObject * args)
   Py_RETURN_NONE;
 }
 
-/// Get the list of nodes discovered by the provided node
-/**
- *  Raises ValueError if pynode is not a node capsule
- *  Raises RuntimeError  if there is an rcl error
- *
- * \param[in] pynode Capsule pointing to the node
- * \return Python list of tuples where each tuple contains the two strings:
- *   the node name and node namespace
- */
 static PyObject *
```

Comment:
even if an impl function, can this still have a docblock ?

Reply:
See https://github.com/ros2/rclpy/pull/538/commits/48651a1cd0202243258952ba6ab0d715adc3b217.